plugins/uefi-capsule/fu-uefi.rs

```diff
@@ -89,3 +89,16 @@ struct FuStructBitmapInfoHeader {
     width: u32le,
     height: u32le,
 }
+
+#[repr(u32le)]
+enum FuAbIndicationsValue {
+    Unknown,
+    Revert,
+    Accept,
```

Comment:
@apalos is this an enum or a bitfield?

Reply:
We don't have unknown , you can jeep it in there if it helps. We either have an accept or a revert capsule

Reply:
I'm also not sure if it makes sense to write the ESRT GUID into that file at the same time -- e.g. imagine the case where you have two esrt entries, one for a ME, and one for a system firmware. They're updated separately, and I think it makes sense to be able to activate either of them individually.